components/VC/Views/VCCardView.tsx

```diff
@@ -102,6 +102,7 @@ export const VCCardView: React.FC<VCItemProps> = props => {
       KEBAB_POPUP={controller.KEBAB_POPUP}
       isVerified={props.vcMetadata.isVerified}
       isInitialLaunch={props.isInitialLaunch}
+      isExpired={props.vcMetadata.isExpired}
```

Comment:
Do we need to send `isExpired` prop explicitly given `vcMetadata` is already passed?